taxi/tests.py

```diff
@@ -1,3 +1,195 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
 from django.test import TestCase
+from django.urls import reverse
 
-# Create your tests here.
+from taxi.forms import (DriverSearchForm,
+                        CarSearchForm,
+                        ManufacturerSearchForm,
+                        DriverCreationForm,
+                        validate_license_number,
+                        DriverLicenseUpdateForm)
+from taxi.models import Driver, Car, Manufacturer
+
+
+class TestSearchForms(TestCase):
```

Comment:
did u read tutorial ? 
https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Testing#:~:text=and%20run%20tests.-,Test%20structure%20overview,-Before%20we%20go